tests/motion/spindlesync-exceeds-maxvel/test-ui.py

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env python
+
+import linuxcnc, hal
+import sys, os, time
+
+
+# Params
+spindle_speed = 2000 # RPM
+z_axis_maxvel = 4 * 60 # IPM
+# Pitch at which Z axis must travel at double maxvel if spindle speed
+# is constant
+max_pitch = float(z_axis_maxvel) / float(spindle_speed) * 2
+
+
+# Set up component
+h = hal.component("python-ui")
+# - Monitor spindle and Z axis speed for debugging
+h.newpin("spindle-speed", hal.HAL_FLOAT, hal.HAL_IN)
+h.newpin("Zvel", hal.HAL_FLOAT, hal.HAL_IN)
+# - Estimated pitch
+h.newpin("pitch", hal.HAL_FLOAT, hal.HAL_IN)
+# - Signal program start/stop
+h.newpin("running", hal.HAL_BIT, hal.HAL_IN)
+h.ready() # mark the component as 'ready'
+os.system("halcmd source ./postgui.hal")
+
+
+# Initialization
+c = linuxcnc.command()
+s = linuxcnc.stat()
+c.state(linuxcnc.STATE_ESTOP_RESET)
+c.state(linuxcnc.STATE_ON)
+c.mode(linuxcnc.MODE_MDI)
+
+# Spindle:  set speed and start
+c.mdi('S%d M3' % spindle_speed)
+    
+# Spindle synchronized motion at 2x Z axis maxvel
+c.mdi('G33 Z-5 K%.3f' % max_pitch)
+
+# While above MDI command runs, take an average of pitch while the Z
+# axis is in motion
+running = True
+timeout = 5
+avg_sum = 0; avg_count = 0
+while timeout > 0:
+    if abs(h['pitch']) > 0.001:
```

Comment:
I tried porting the spindlesync-exceeds-maxvel test to the latest master, and ran into a problem with h['pitch'] never being above 0.001.  Where is it supposed to be set?